bin/run-browserstack-tests.js

```diff
@@ -1,71 +1,39 @@
-/* eslint-disable no-console */
+/* eslint-disable no-console, node/no-unsupported-features */
 
-var RSVP = require('rsvp');
-var spawn = require('child_process').spawn;
+var execa = require('execa');
```

Comment:
Seems _generally_ odd to me to use `async` / `await` **and** `var` in the same file... 😹

Reply:
agreed, changed to `const`